File: Codes/1450-number-of-students-doing-homework-at-a-given-time_20240901.go
/*
 * @Author: LetMeFly
 * @Date: 2024-09-01 14:56:16
 * @LastEditors: LetMeFly
 * @LastEditTime: 2024-09-01 15:02:25
 */
package main

func busyStudent(startTime []int, endTime []int, queryTime int) int {
    ans := 0
    for i := range startTime {
        if startTime[i] <= queryTime && queryTime <= endTime[i] {
            ans++
        }
    }
    return ans
}

File: Codes/1450-number-of-students-doing-homework-at-a-given-time_20240901.cpp
/*
 * @Author: LetMeFly
 * @Date: 2024-09-01 14:55:50
 * @LastEditors: LetMeFly
 * @LastEditTime: 2024-09-01 14:57:24
 */
#ifdef _WIN32
#include "_[1,2]toVector.h"
#endif

class Solution {
public:
    int busyStudent(vector<int>& startTime, vector<int>& endTime, int queryTime) {
        int ans = 0;
        for (int i = startTime.size() - 1; i >= 0; i--) {
            ans += startTime[i] <= queryTime && endTime[i] >= queryTime;
        }
        return ans;
    }
};